app/customer.py

```diff
@@ -0,0 +1,66 @@
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list[int],
+            money: int | float
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.home_location = location
+        self.location = self.home_location
+        self.money = money
+        self.trip_costs = {}
+        self.chosen_shop = None
+        self.spent_money = 0
+
+    def amount_of_money_at_start(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def cost_of_all_trips(
+            self,
+            stores: list,
+            fuel_consumption: float,
+            fuel_price: float

```

Comment:
The parameter 'stores' should have a more descriptive type hint. Instead of 'list', consider using 'list[dict]' to indicate that it is a list of dictionaries.